node.gyp

```diff
@@ -139,6 +139,7 @@
         'src/node_main.cc',
         'src/node_os.cc',
         'src/node_revert.cc',
+        'src/node_report.cc',
```

Comment:
can you also add the header file down below? It makes it easier when using Xcode or other IDEs